contracts/token/mocks/ZTokenMock.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.26;
+
+import { ZToken } from "@zero-tech/z-token/contracts/ZToken.sol";
+
+
+contract ZTokenMock is ZToken {
```

Comment:
As a mock it would be helpful to have a public `mint` function, similar to what `MeowTokenMock.sol` had already.